### PR TITLE
Updating the roadmap page in the documentation to make it more readable

### PR DESCRIPTION
**Pull request by fcorti**
## Hey, I just made a Pull Request!

Updating the roadmap page in the documentation to make it more readable

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/overview/roadmap.md [hunk 1]
```diff
@@ -6,55 +6,17 @@ description: Roadmap of Backstage
...
 
-- Upcoming enhancements and benefits,
-- Planning contributions and support,
-- Planning the project’s adoption,
-- Understanding what things are coming soon,
-- Avoiding duplication of work
+The Backstage roadmap lays out both [“what’s next”](#whats-next) and [“future work”](#future-work). With "next" we mean features planned for release within the ongoing quarter from January through March 2022. With "future" we mean features in the radar, but not yet scheduled.
 
-### How to influence the roadmap
+| What's next | [Future work](#future-work) |
```
Comment (Rugvip) [inline docs/overview/roadmap.md:13]: Feeling that "What's next" should get a link here too 😅

**Review by Rugvip — APPROVED** [review]
🎉, pre-approved

We'll probably try out https://github.com/backstage/backstage/pull/9181 to avoid having doc changes run into as many formatting issues, but for now you'll need to `yarn prettier --write docs/overview/roadmap.md` or something like that to fix the formatting

File: docs/overview/roadmap.md (hunk 1 above)
Comment (fcorti) [reply docs/overview/roadmap.md]: Yep, added!
Thanks

**Comment by fcorti** [comment]
@Rugvip I run the prettier and pushed the updated version in this PR.
Thanks.

**Review by jhaals — APPROVED** [review]
Nice work 🙌

**Pull request merged**